index.go

```diff
@@ -242,15 +267,40 @@ func LoadIndexXML(r io.Reader) (*Index, error) {
 	return &index, nil
 }
 
-func (a *App) CurApk() *Apk {
+func (a *App) ApksByVName(vname string) ApkList {
+	var apks ApkList
 	for i := range a.Apks {
-		apk := a.Apks[i]
-		if a.CVCode >= apk.VCode {
-			return &apk
+		if vname == a.Apks[i].VName {
+			apks = append(apks, a.Apks[i])
 		}
 	}
-	if len(a.Apks) > 0 {
-		return &a.Apks[0]
+	return apks
+}
+
+func (a *App) SuggestedApks() ApkList {
```

Comment:
Why are you using `ApkList` instead of `[]Apk` here? IIRC `ApkList` is just used to implement `sort.Interface`, it shouldn't be used as a general rule.